test/unit/event/send-ack-error-event.test.js

```diff
@@ -17,31 +17,31 @@ jest.mock('ffc-pay-event-publisher', () => {
   }
 })
 jest.mock('../../../app/processing/get-payment-request-by-invoice-frn')
-const getPaymentSchemeByInvoiceAndFrn = require('../../../app/processing/get-payment-request-by-invoice-frn')
+const { getPaymentRequestByInvoiceAndFrn } = require('../../../app/processing/get-payment-request-by-invoice-frn')
 jest.mock('../../../app/config')
-const config = require('../../../app/config')
+const { processingConfig, messageConfig } = require('../../../app/config')
 const { PAYMENT_DAX_REJECTED, PAYMENT_INVALID_BANK } = require('../../../app/constants/events')
 const { SOURCE } = require('../../../app/constants/source')
-const sendAcknowledgementErrorEvent = require('../../../app/event/send-acknowledgement-error-event')
+const { sendAcknowledgementErrorEvent } = require('../../../app/event/send-acknowledgement-error-event')
 
 let paymentRequest
 let acknowledgement
 let holdCategoryNameDR
 let holdCategoryNameBAA
-const frn = require('../../mocks/frn')
+const frn = require('../../mocks/values/frn')
 
 beforeEach(() => {
-  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-request')))
-  acknowledgement = JSON.parse(JSON.stringify(require('../../mocks/acknowledgement-error')))
+  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-requests/payment-request')))
+  acknowledgement = JSON.parse(JSON.stringify(require('../../mocks/acknowledgement')))
   holdCategoryNameDR = JSON.parse(JSON.stringify(require('../../../app/constants/hold-categories-names'))).DAX_REJECTION
   holdCategoryNameBAA = JSON.parse(JSON.stringify(require('../../../app/constants/hold-categories-names'))).BANK_ACCOUNT_ANOMALY
 
-  getPaymentSchemeByInvoiceAndFrn.mockResolvedValue(paymentRequest)
+  getPaymentRequestByInvoiceAndFrn.mockResolvedValue(paymentRequest)
 
-  config.useV1Events = true
-  config.useV2Events = true
-  config.eventTopic = 'v1-events'
-  config.eventsTopic = 'v2-events'
+  processingConfig.useV1Events = true
+  processingConfig.useV2Events = true
+  messageConfig.eventTopic = 'v1-events'
+  messageConfig.eventsTopic = 'v2-events'
 })
```

Comment:
Should the before and after each not be defined In the describe?

Reply:
As before :)